docs/pages/guides/tutorials/deploy-cassandra-with-statefulset.md

```diff
@@ -0,0 +1,129 @@
+
+### Prerequisites
+
+- Basic understanding of Kubernetes concepts.
+- Access to the _Meshery Playground_. If you don't have an account, sign up at [Meshery Playground](https://play.meshery.io/).
+
+### Lab Scenario
+
+- Import the Apache Service and Apache StatefulSet manifest files into Meshery Playground.
+
+- Create a Cassandra ring using the visual tools provided by Meshery.
+
+- Deploy these resources on the playground.
+
+In this lab, you will import the Service and StatefulSet manifest files into Meshery Playground. You will visualize these Kubernetes resources and Cassandra Ring using the visual tools provided by Meshery. Finally, you will deploy these resources on the Playground.
+
+### Objective
+
+Learn how to import manifest files, visualize Kubernetes resources, create new resource components, and deploy the application using Meshery Playground.
+
+### Steps
+
+#### Download the Kubernetes Configuration Files
+
+Go ahead and download these yaml files [cassandra-service.yaml](https://kubernetes.io/examples/application/cassandra/cassandra-service.yaml) and [cassandra-statefulset.yaml](https://kubernetes.io/examples/application/cassandra/cassandra-statefulset.yaml)
+
+These YAML files contain the Cassandra Service and Cassandra StatefulSet manifests
+
+#### Access Meshery Playground
+
+1. Log in to the [Meshery Playground](https://meshery.layer5.io/) using your credentials. On successful login, you should be at the dashboard. Press the **X** on the _Where do you want to start?_ popup to close it (if required).
+
+2. Click **Explore** in the Cloud Native Playground tile to navigate to _MeshMap_
+
+#### Import the Files to Meshery Playground
+
+1. In the left sidebar, click on the upward arrow symbol(import icon) to import the designs into Meshery.
+
+2. In the modal that appears:
+   - Enter a name for your design in the "Design File Name" field (e.g.`cassandra-service`).
+
+   - Select `Kubernetes Manifest` from the "Design Type" dropdown menu.
+
+        ![Import modal](./screenshots/cassandra-1.png)
+
+   - Choose `File Upload` for the upload method, and select the file you just downloaded.
+
+   - Then, click on `Import`
+
+        ![Import cassandra-service](./screenshots/cassandra-2.png)
+
+3. Under the "Designs" tab, you will see that we have successfully imported the `cassandra-service` design.
+
+   When you click on the names of the designs on the `Designs` tab, you can see the visual representations of the various Kubernetes resources and their relationships on the canvas.
+
+    ![Imported designs on canvas](./screenshots/cassandra-3.png)
+
+4. Now, follow the same steps to import the `cassandra-statefulset` file.
+
+    ![cassandra-statefulset](./screenshots/cassandra-4.png)
+
+#### Merging the Designs
```

Comment:
It's nice to see a design merge being performed. This action isn't always included in tutorials, but is a power capability to showoff and familiarize users with.